devel/libcoverart/Portfile

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+PortGroup           github 1.0
+
+github.setup        metabrainz libcoverart 1.0.0 release-
+github.tarball_from releases
+categories          devel
+platforms           darwin
+maintainers         nomaintainer
+license             LGPL-2.1+
+
+description         C/C++ library for accessing the MusicBrainz Cover Art Archive
+
+long_description    libcoverart is a C/C++ library for accessing the MusicBrainz Cover Art Archive
+
+homepage            https://musicbrainz.org/doc/libcoverart
+
+checksums           rmd160  2c3d15b3399b16b842cde85fa02401caf72e4cdb \
+                    sha256  5d9d4a4c46f35754e016affab83957961a364d7a4c7d64447f73bff22a300f44 \
+                    size    57933
+
+depends_lib         \
+                    port:neon \
+                    port:jansson \
+                    port:libxml2
+
+depends_build       \
+                    port:pkgconfig
+
+PortGroup           cmake 1.0
```

Comment:
This PortGroup line belongs at the top with the other PortGroup line. We usually sort PortGroup lines alphabetically.

Is there a reason why the cmake 1.1 portgroup did not work? If not, please use that instead. It does out of source builds by default and has other improvements over cmake 1.0.

Reply:
I had the PortGroup line here because I didn't know about needing to use `depends_build-append` above.

PortGroup cmake 1.1 gives this error:
`make[2]: *** No rule to make target `src/*.inc', needed by `src/caa_c.cc'.  Stop.`

PortGroup cmake 1.0 builds without errors.

Reply:
Does it work if you use the cmake 1.1 portgroup and add the line `cmake.out_of_source no`? If so, then this software's build system does not support out of source builds, and that would be a bug worth reporting to its developers.

Reply:
This software is from 2014. I found three commits made after its release that try to fix out of source builds:

https://github.com/metabrainz/libcoverart/commit/7d6019355037b5fc2445ad53c117f41ad703013c
https://github.com/metabrainz/libcoverart/commit/17a7f2e79cefe74dd028b26e174030b4f6d08fba
https://github.com/metabrainz/libcoverart/commit/ede742fa4cba8711deec1da818b419add9211e00

Maybe applying those diffs to the port would fix the problem.

Reply:
> Does it work if you use the cmake 1.1 portgroup and add the line `cmake.out_of_source no`?

Yes. It does work with this.

I will try out the commits you found.

Reply:
I tried the commits you found, but no luck for `PortGroup cmake 1.1` without `cmake.out_of_source no`
I still get this error even with the commits as patches:
`make[2]: *** No rule to make target src/*.inc', needed by src/caa_c.cc'. Stop.`